Examples/Run/Reconstruction/RecTruthTracks.cpp

```diff
@@ -102,8 +102,11 @@ int main(int argc, char* argv[]) {
 
   // Create smeared measurements
   HitSmearing::Config hitSmearingCfg;
-  hitSmearingCfg.inputSimulatedHits = clusterReaderCfg.outputSimulatedHits;
+  hitSmearingCfg.inputSimHits = clusterReaderCfg.outputSimHits;
+  hitSmearingCfg.outputMeasurements = "measurements";
   hitSmearingCfg.outputSourceLinks = "sourcelinks";
+  hitSmearingCfg.outputHitParticlesMap = "digi_hit_particles_map";
```

Comment:
For consistency, could you move the `TruthSeedSelector` behind the `HitSmearing` and use the`hitSmearingCfg.outputHitParticlesMap` for the `particleSelectorCfg.inputHitParticlesMap` as well? Something similar could be done for the `RecCKFTracks`.

Reply:
Yes, that is much clearer. Especially, since I expect that we will not use the planar cluster reader in the future for this.

Reply:
The output from the cluster reader is now not used anymore.